index.html

```diff
@@ -49,5 +52,170 @@ <h3>I am a Software Developer</h3>
 		<img src="images/Illustration.png" alt="shapes background image" class="png">
 	</section>
 
+	<section class="works">
+		<div class="container">
+			<h1>My Recent Works</h1>
+			<div class="project">
+				<img src="images/image 1.png" alt="A yoga lady">
+				<h2>Multi-Post Stories</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy 
+				   text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+				   <ul class="lang">
+					<li><a href="">css</a></li>
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+			</div>
+			<button class="btn"><a href="">See Project</a></button>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
```

Comment:
@Mikey1-nuel  Contrary to the above section, the background color of the tags is not white ie. `background: rgba(255, 255, 255);` but it is 
`background: rgba(255, 255, 255, 0.24);`

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219881328-a41d912f-89aa-40fd-b26d-c32e28e9452e.png)| ![image](https://user-images.githubusercontent.com/61361037/219881308-5acc922e-984a-4f92-a005-b6d0f68d48e9.png)|

Reply:
Add the same background color to the next work cards too...